### PR TITLE
Edit support articles

### PR DESCRIPTION
**Pull request by ebrett**
### Context

Follow up issues for [HFEYP-553](https://dfedigital.atlassian.net/browse/HFEYP-553)
and [HFEYP-548](https://dfedigital.atlassian.net/browse/HFEYP-548) Edit and create of Articles

[HFEYP-580](https://dfedigital.atlassian.net/browse/HFEYP-580)
[HFEYP-581](https://dfedigital.atlassian.net/browse/HFEYP-581)
[HFEYP-582](https://dfedigital.atlassian.net/browse/HFEYP-582)

[HFEYP-574](https://dfedigital.atlassian.net/browse/HFEYP-574)
[HFEYP-575](https://dfedigital.atlassian.net/browse/HFEYP-575)
[HFEYP-576](https://dfedigital.atlassian.net/browse/HFEYP-576)
[HFEYP-577](https://dfedigital.atlassian.net/browse/HFEYP-577)
[HFEYP-578](https://dfedigital.atlassian.net/browse/HFEYP-578)
[HFEYP-579](https://dfedigital.atlassian.net/browse/HFEYP-579)

### Changes proposed in this pull request

The image upload are always shown now, so image can be replaced.

### CODE REVIEW

File: spec/features/admin/admin_articles_spec.rb [hunk 1]
```diff
@@ -14,9 +9,10 @@
   end
 
   context "with session" do
-    let(:editor) { User.find_by(email: "pwhite@education.gov.uk") }
     scenario "visits article admin" do
-      login_as(editor)
+      create :article, title: "A typical support article", description: "And it's description", markdown: "For score and seven years ago..."
+      create :article, title: "Another support article", description: "And still another description", markdown: "How now brown cow?"
```
Comment (guy-roberts) [inline spec/features/admin/admin_articles_spec.rb:14]: Wish I could have persuaded you of the usefulness of FactoryGirl, it saves a lot of repetition, and any differences can be done as traits

File: spec/features/admin/admin_articles_spec.rb [hunk 2]
```diff
@@ -31,5 +27,15 @@
...
+
+    scenario "delete listed article" do
+      create :article, title: "Delete me"
+      login_as(create(:editor))
+      visit "/admin/articles"
+      expect(page).to have_text("Delete me")
+      click_on "Delete"
+      accept_confirm "Are you sure you want to delete?"
+      expect(page).to_not have_text("Delete me")
+    end
```
Comment (guy-roberts) [inline spec/features/admin/admin_articles_spec.rb:39]: Delete me sounds a bit informal !

**Review by guy-roberts — APPROVED** [review]

File: spec/features/admin/admin_articles_spec.rb (hunk 1 above)
Comment (ebrett) [reply spec/features/admin/admin_articles_spec.rb:14]: I am using FactoryGirl - the class name is not required.

File: spec/features/admin/admin_articles_spec.rb (hunk 2 above)
Comment (ebrett) [reply spec/features/admin/admin_articles_spec.rb:39]: Delete me, sir?

File: spec/features/admin/admin_articles_spec.rb (hunk 1 above)
Comment (ebrett) [reply spec/features/admin/admin_articles_spec.rb]: Better?

**Review by guy-roberts — APPROVED** [review]

**Pull request merged**